tools/sklearn/main_macros.xml

```diff
@@ -825,10 +825,114 @@
+        <expand macro="random_state" />
+      </when>
+      <when value="RepeatedStratifiedKFold">
+        <expand macro="cv_n_splits" value="5"/>
+        <param argument="n_repeats" type="integer" value="10" label="n_repeats" help="Number of times cross-validator needs to be repeated." />
+        <expand macro="random_state" />
+      </when>
+      <when value="ShuffleSplit">
+        <expand macro="cv_n_splits" value="10" help="Number of re-shuffling and splitting iterations."/>
+        <expand macro="cv_test_size" value="0.1" />
+        <expand macro="random_state"/>
+      </when>
+      <when value="StratifiedShuffleSplit">
+        <expand macro="cv_n_splits" value="10" help="Number of re-shuffling and splitting iterations."/>
+        <expand macro="cv_test_size" value="0.1" />
+        <expand macro="random_state"/>
+      </when>
+      <when value="TimeSeriesSplit">
+        <expand macro="cv_n_splits"/>
+        <param argument="max_train_size" type="integer" value="" optional="true" label="max_train_size" help="Maximum size for a single training set." />
+      </when>
+      <when value="PredefinedSplit">
+        <param argument="test_fold" type="text" value="" area="true" label="test_fold" help="List, e.g., [0, 1, -1, 1], represents two test sets, [X[0]] and [X[1], X[3]], X[2] is excluded from any test set due to '-1'.">
+          <sanitizer>
+            <valid initial="default">
+              <add value="["/>
+              <add value="]"/>
+            </valid>
+          </sanitizer>
+        </param>
+      </when>
+      <when value="GroupKFold">
+        <expand macro="cv_n_splits"/>
+        <expand macro="cv_groups" />
+      </when>
+      <when value="GroupShuffleSplit">
+        <expand macro="cv_n_splits" value="5"/>
+        <expand macro="cv_test_size"/>
+        <expand macro="random_state"/>
+        <expand macro="cv_groups"/>
+      </when>
+      <when value="LeaveOneGroupOut">
+        <expand macro="cv_groups"/>
+      </when>
+      <when value="LeavePGroupsOut">
+        <param argument="n_groups" type="integer" value="" label="n_groups" help="Number of groups (p) to leave out in the test split." />
+        <expand macro="cv_groups"/>
+      </when>
+    </conditional>
+  </xml>
+
+  <xml name="cv_n_splits" token_value="3" token_help="Number of folds. Muast be at least 2.">
+    <param argument="n_splits" type="integer" value="@VALUE@" label="n_splits" help="@HELP@"/>
+  </xml>
+
+  <xml name="cv_shuffle">
+    <param argument="shuffle" type="boolean" optional="true" truevalue="booltrue" falsevalue="boolfalse" checked="false" label="shuffle" />
+  </xml>
+
+  <xml name="cv_test_size" token_value="0.2">
+    <param argument="test_size" type="float" value="@VALUE@" label="test_size" help="0.0-1.0, proportion of the dataset to include in the test split; >1, integer only, the absolute number of test samples "/>
```

Comment:
remove `0.0-1.0` and use `min="0.0" max="1.0"`

Reply:
We can add `min="0.0"`. `test_size` takes both float, 0.0-1.0, and integer, greater than 1.0. When integer, represent the number of instances in tests.